tests/spoke/spoke_suite_test.go

```diff
@@ -185,35 +196,48 @@ var _ = BeforeSuite(func() {
 	}).SetupWithManager(k8sManager)
 	Expect(err).ToNot(HaveOccurred())
 
+	// spokeServiceImportEventRecorder := mevents.NewEventRecorder(k8sClient, k8sManager.GetScheme(), ossEvents.EventsMap, mevents.EventRecorderOptions{
+	// 	Cluster:   hub.ClusterName,
+	// 	Project:   PROJECT_NS,
+	// 	Component: "serviceimport-operator",
+	// 	Namespace: CONTROL_PLANE_NS,
+	// })
 	err = (&serviceimport.Reconciler{
-		Client: k8sManager.GetClient(),
-		Scheme: k8sManager.GetScheme(),
-		Log:    ctrl.Log.WithName("SvcImTest"),
-		EventRecorder: &events.EventRecorder{
-			Recorder: &record.FakeRecorder{},
-		},
+		Client:        k8sManager.GetClient(),
+		Scheme:        k8sManager.GetScheme(),
+		Log:           ctrl.Log.WithName("SvcImTest"),
+		EventRecorder: &spokeSliceEventRecorder,
 	}).Setup(k8sManager, mf)
 	Expect(err).ToNot(HaveOccurred())
 
+	// spokeNamespaceEventRecorder := mevents.NewEventRecorder(k8sClient, k8sManager.GetScheme(), ossEvents.EventsMap, mevents.EventRecorderOptions{
+	// 	Cluster:   hub.ClusterName,
+	// 	Project:   PROJECT_NS,
+	// 	Component: "namespace_reconciler",
+	// 	Namespace: CONTROL_PLANE_NS,
+	// })
 	err = (&namespace.Reconciler{
-		Client: k8sManager.GetClient(),
-		Scheme: k8sManager.GetScheme(),
-		Log:    ctrl.Log.WithName("NamespaceTest"),
-		EventRecorder: &events.EventRecorder{
-			Recorder: &record.FakeRecorder{},
-		},
+		Client:        k8sManager.GetClient(),
+		Scheme:        k8sManager.GetScheme(),
+		Log:           ctrl.Log.WithName("NamespaceTest"),
+		EventRecorder: &spokeSliceEventRecorder,
 		Hubclient: &hub.HubClientConfig{
 			Client: k8sClient,
 		},
 	}).SetupWithManager(k8sManager)
 	Expect(err).ToNot(HaveOccurred())
 
-	netpolEventRecorder := events.NewEventRecorder(k8sManager.GetEventRecorderFor("networkpolicy-controller"))
+	// netpolEventRecorder := mevents.NewEventRecorder(k8sClient, k8sManager.GetScheme(), ossEvents.EventsMap, mevents.EventRecorderOptions{
+	// 	Cluster:   hub.ClusterName,
+	// 	Project:   PROJECT_NS,
+	// 	Component: "networkpolicy-reconciler",
+	// 	Namespace: CONTROL_PLANE_NS,
+	// })
 	err = (&networkpolicy.NetpolReconciler{
 		Client:        k8sManager.GetClient(),
 		Log:           ctrl.Log.WithName("controllers").WithName("networkpolicy"),
 		Scheme:        k8sManager.GetScheme(),
-		EventRecorder: netpolEventRecorder,
+		EventRecorder: &spokeSliceEventRecorder,
```

Comment:
spokeSliceEventRecorder to workerSliceEventRecorder

Reply:
done